policy/policy.go

```diff
@@ -45,14 +45,14 @@ var (
 	// EmptyVersionedPolicies represents an empty VersionPolicies
 	EmptyVersionedPolicies VersionedPolicies
 
-	errNilPolicySchema = errors.New("nil policy schema")
-
-	// defaultPolicies are the default policies
+	// DefaultPolicies are the default policies
 	// TODO(xichen): possibly make this dynamically configurable in the future
-	defaultPolicies = []Policy{
+	DefaultPolicies = []Policy{
```

Comment:
I was using them in tests for the encoder, but I can get them through a VersionedPolicies object that is initialized with the default policies so I'll do that instead.